app/assets/stylesheets/farmbot.css.scss

```diff
@@ -98,6 +101,46 @@ $brown:          #CA8;
   }
 }
 
+.designer-crop-info {
+  .crop-drag-info-image {
+    max-width: 98%;
+    padding: 0 auto 0 auto;
+    background-color: $translucent;
+  }
+  .crop-info-overlay {
+    position: relative;
```

Comment:
Properties should be ordered align-items, background-color, bottom, color, display, font-style, height, justify-content, max-width, position, z-index